packages/driver/cypress/integration/e2e/multi-domain/basic_login_spec.ts

```diff
@@ -0,0 +1,150 @@
+// @ts-ignore / session support is needed for visiting about:blank between tests
+describe('basic login', { experimentalSessionSupport: true, experimentalMultiDomain: true }, () => {
+  // Diagram A
+  describe('visit primary first', () => {
+    it('logs in with idp redirect', () => {
+      cy.visit('/fixtures/auth/index.html') // Establishes Primary Domain
```

Comment:
I believe the primary domain has already been established by the baseUrl.

Reply:
Yeah, i think so, but i will try unsetting here, either baseurl or the first visit should establish the primary domain